components/event/speakerCard.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <article
+    class="w-full p-4 mb-4 overflow-hidden border-2 border-gray-200 shadow-md"
+  >
+    <div class="flex flex-col sm:flex-row">
+      <div class="flex flex-col">
+        <img
+          :src="filterImageQuality(speaker.image, '65')"
+          :alt="speaker.name"
+          class="w-48 mx-auto"
+        />
+        <div class="mt-4 text-center flex justify-center">
+          <a
+            v-if="speaker.github.url"
+            :href="speaker.github.url"
+            class="p-2 icon"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <font-awesome-icon
+              :icon="['fab', 'github']"
+              class="w-4 h-4 fill-current text-mtl-primary"
+            />
+          </a>
+          <a
+            v-if="speaker.linkedin.url"
+            :href="speaker.linkedin.url"
+            class="p-2 icon"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <font-awesome-icon
+              :icon="['fab', 'linkedin']"
+              class="w-4 h-4 fill-current text-mtl-primary"
+            />
+          </a>
+        </div>
+      </div>
+      <div v-if="speaker.description" class="flex flex-col w-full ml-4">
+        <text-description :text="speaker.description" />
+      </div>
+    </div>
+
+    <div v-if="speaker.video.url" class="mt-4">
+      <h3 class="text-mtl-infos">Video:</h3>
+      <div class="relative mt-2 embed-responsive aspect-ratio-16/9">
+        <iframe
+          :src="speaker.video.url"
+          frameborder="0"
+          allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen
+          class="absolute w-full h-full pin"
+        ></iframe>
+      </div>
+    </div>
+
+    <div v-if="speaker.slides.url" class="mt-8">
+      <span class="text-mtl-infos"> Slides: </span>
+      <a
+        :href="speaker.slides.url"
+        target="_blank"
+        class="font-bold underline text-mtl-primary "
+        >{{ $t('click_here') }}</a
+      >
+    </div>
```

Comment:
I know this is a bit out of scope, but let's use this refactor to improve this. We should [avoid "Click here" as much as possible](https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/) since it's not a good practice. 

Can you put instead a simple link that reads:  
EN: `View slides`
FR: `Voir les diapositives` 

If I understand well you'll have to add the translation here: 

https://github.com/vuemontreal/vuemontreal/blob/928991b80e8cdce6bd76cba9e95f2ec6936a158b/configs/i18n.js#L11-L26

Please also remove / clean up the `click_here` as it's not used anywhere else.


```suggestion
    <div v-if="speaker.slides.url" class="mt-8">
      <a
        :href="speaker.slides.url"
        target="_blank"
        class="font-bold underline text-mtl-primary "
        >{{ $t('view_slides') }}</a
      >
    </div>
```

Reply:
good point! addressed in 38a07d6

Reply:
Perfect! 💯